contracts/solution/src/test.rs

```diff
@@ -100,7 +100,7 @@ pub fn fca00c_budget() {
 
     // We reset the budget here so that we *only* count the budget for your
     // contract's `solve()` function. Everything else we've done so far is free!
-    env.budget().reset();
+    env.budget().reset_unlimited();
```

Comment:
As we are in the 'close-to-reality test' we might want to replace this budget limit with the actual limit used on the backend.

Reply:
🤔 I agree. What would that look like?